ray-operator/controllers/ray/common/pod.go

```diff
@@ -706,11 +711,16 @@ func concatenateContainerCommand(nodeType rayiov1alpha1.RayNodeType, rayStartPar
 
 func convertParamMap(rayStartParams map[string]string) (s string) {
 	flags := new(bytes.Buffer)
-	nonFlagParams := []string{"log-color", "include-dashboard"}
+	// NonFlagParams with a value of true or false.
+	specialNonFlagParams := []string{"log-color", "include-dashboard"}
 	for k, v := range rayStartParams {
-		if strings.ToLower(v) == "true" && !utils.Contains(nonFlagParams, k) {
-			fmt.Fprintf(flags, " --%s ", k)
+		if utils.Contains([]string{"true", "false"}, strings.ToLower(v)) && !utils.Contains(specialNonFlagParams, k) {
```

Comment:
handle the case if user sets rayStartParams["XXX"] = "false". XXX can be `block` option.